From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Qais Yousef <qais.yousef@arm.com>
Date: Mon, 21 Sep 2020 15:04:13 +0100
Subject: ANDROID: arm64: Enable KVM for Asym AArch32

Now KVM better handles the asym aarch32 situation. Allow both configs to
coexist.

Bug: 168847043
Reason: Needed for bringup. Revert when upstream patch is available
Signed-off-by: Qais Yousef <qais.yousef@arm.com>
Change-Id: I62d8fd58ac950846f18cf2cb848a3b59922e9bd6
---
 arch/arm64/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -1897,7 +1897,7 @@ endmenu
 
 config ASYMMETRIC_AARCH32
 	bool "Allow support for asymmetric AArch32 support"
-	depends on COMPAT && EXPERT && !KVM
+	depends on COMPAT && EXPERT
 	help
 	  Enable this option to allow support for asymmetric AArch32 EL0
 	  CPU configurations. Once the AArch32 EL0 support is detected
